.github/workflows/ruby.yml

```
@@ -5,14 +5,16 @@ on: [push, pull_request]
 jobs:
   test:
     runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        ruby: [ '2.4', '2.5', '2.6', '2.7', '3.0' ]
```

Honestly, as I remember, it can be simplify with native YAML syntax, but as you wish:

```suggestion
        ruby:
          - 2.4
          - 2.5
          - 2.6
          - 2.7
          - 3.0
```

---

Either way should be fine, though I do prefer your way @AlexWayfer